app/src/main/java/com/kickstarter/mock/factories/RewardFactory.java

```diff
@@ -16,6 +16,7 @@ private RewardFactory() {}
   public static @NonNull Reward addOn() {
     return reward().toBuilder()
             .isAddOn(true)
+            .limit(10)
```

Comment:
what is the limit field and why is it 10?

Reply:
Is the addOns Limit, it wont go over 10. 
RewardFactory holds mocks for tests, I added the limit for the addOn object because there are some unit test checking availability for addOns using the limit.